chart/values.yaml

```diff
@@ -18,7 +18,7 @@ distributor:
   image:
     repository: docker.io/keptn/distributor  # Container Image Name
     pullPolicy: IfNotPresent                 # Kubernetes Image Pull Policy
-    tag: "0.13.4"                             # Container Tag
+    tag: "0.14.1"                             # Container Tag
```

Comment:
Please use at least 0.14.2 by default, keptn 0.14.1 has a well known bug in the aggregation of deploymentURIs and frankly I would try to avoid any reference to that version (even if the distributor is not involved)